app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsAppearanceScreen.kt

```diff
@@ -84,6 +100,7 @@ object SettingsAppearanceScreen : SearchableSettings {
                         )
                     }
                 },
+                *customPreferenceItem.toTypedArray(),
```

Comment:
Wouldn't it be better to use `PreferenceGroup` instead of trying to change a persistent list?